### PR TITLE
Merge Bugfix/removed-old-navigationservice-reference into develop

### PR DESCRIPTION
**Pull request by Pantheas**
Removed old `IViewModelNavigationService` reference from XML comments and renamed `SetRoot` to `SetRootAsync`

### CODE REVIEW

File: src/Navigation/WPF/Navigation.WPF/Service/NavigationService.cs [hunk 1]
```diff
@@ -80,8 +83,15 @@ protected NavigationStackEntry Current
 
 
         public IViewModel RootViewModel => Root?.ViewModel;
+
         public IViewModel CurrentViewModel => Current?.ViewModel;
 
+
+        public bool CanGoBack => BackStack?.Any() == true;
```
Comment (Pantheas) [inline src/Navigation/WPF/Navigation.WPF/Service/NavigationService.cs:90]: Codacy Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5674091)

**Comment by Pantheas** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5674091&bid=18576055)

File: src/MVVM/ViewModels/Navigation/ViewModelBase.cs [hunk 2]
```diff
@@ -58,7 +58,7 @@ await closeViewModelTask
         }
 
         /// <summary>
-        /// Closes this ViewModel and informs the parent using the IViewModelNavigationService (if registered)
+        /// Closes this ViewModel and informs the parent using the <see cref="CodeMonkeys.Navigation.INavigationService"/> (if registered)
```
Comment (Pantheas) [inline src/MVVM/ViewModels/Navigation/ViewModelBase.cs:61]: Codacy Issue found: [Split this 138 characters long line (which is greater than 130 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5674091)

**Comment by Pantheas** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5674091&bid=18576055)

File: src/MVVM/ViewModels/Navigation/ViewModelBase.cs (hunk 2 above)
Comment (Pantheas) [inline src/MVVM/ViewModels/Navigation/ViewModelBase.cs:61]: Codacy Issue found: [Split this 138 characters long line (which is greater than 130 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5674091)

File: src/Navigation/WPF/Navigation.WPF/Service/NavigationService.cs (hunk 1 above)
Comment (Pantheas) [inline src/Navigation/WPF/Navigation.WPF/Service/NavigationService.cs:90]: Codacy Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5674091)

**Review by LegendaryB — APPROVED** [review]

**Pull request merged**